README.md

```diff
@@ -54,6 +54,14 @@ pyenv local spokestack
 pip install -r requirements.txt
 ```
 
+### Build Extensions
+
+When installing from source, you will want to build the Cython extensions.
```

Comment:
It's not necessarily clear that you have to install to run the tests. This might be a better fit for a setup section in the CONTRIBUTING guide or a more general "development" section in the readme.

As it is, this section mixes setup for use (`pip install`) with setup for development on the library itself, which are two pretty different use cases.

You'll also want to add a note about the submodules; you have to run `git submodule update --init` or the equivalent before the extensions can be built.

Reply:
This sounds good. I will move the development specific items to CONTRIBUTING

Reply:
I did some rearranging also and put `pip install spokestack` at the very top.